resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -216,6 +216,35 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <!-- Triggers scalatest plugin in the integration-test phase instead of
+             the test phase, so that test jobs are copied over beforehand.
+             See copy-test-spark-jobs execution of maven-dependency-plugin above. -->
+        <groupId>org.scalatest</groupId>
+        <artifactId>scalatest-maven-plugin</artifactId>
+        <configuration>...</configuration>
```

Comment:
Ah. Turned out they aren't needed. I got this piece from [a scalatest user form](https://groups.google.com/forum/#!topic/scalatest-users/b1ME5zp-7VQ) and the three dots were there just to indicate omission of text.

I am surprised that the presence of three dots in the config did not break maven. Thanks for pushing me to look at this. I'll remove them in the next patch.